boilerplate/app/screens/DemoPodcastListScreen.tsx

```diff
@@ -77,28 +91,126 @@ const EpisodeCard = observer(function EpisodeCard({
   onPressFavorite: () => void
   isFavorite: boolean
 }) {
+  const liked = useSharedValue(isFavorite ? 1 : 0)
+
+  // Grey heart
+  const animatedLikeButtonStyles = useAnimatedStyle(() => {
+    return {
+      transform: [
+        {
+          scale: interpolate(liked.value, [0, 1], [1, 0], Extrapolate.EXTEND),
+        },
+      ],
+      opacity: interpolate(liked.value, [0, 1], [1, 0], Extrapolate.CLAMP),
+    }
+  })
+
+  // Pink heart
+  const animatedUnlikeButtonStyles = useAnimatedStyle(() => {
+    return {
+      transform: [
+        {
+          scale: liked.value,
+        },
+      ],
+      opacity: liked.value,
+    }
+  })
+
+  /**
+   * Android has a "longpress" accessibility action. iOS does not, so we just have to use a hint.
+   * @see https://reactnative.dev/docs/accessibility#accessibilityactions
+   */
+  const accessibilityHintProps = useMemo(
+    () =>
+      Platform.select<AccessibilityProps>({
+        ios: {
+          accessibilityHint: translate("demoPodcastListScreen.accessibility.cardHint", {
+            action: isFavorite ? "unfavorite" : "favorite",
+          }),
+        },
+        android: {
+          accessibilityLabel: episode.title,
+          accessibilityActions: [
+            {
+              name: "longpress",
+              label: translate("demoPodcastListScreen.accessibility.favoriteAction"),
+            },
+          ],
+          onAccessibilityAction: ({ nativeEvent }) => {
+            if (nativeEvent.actionName === "longpress") {
+              handlePressFavorite()
+            }
+          },
+        },
+      }),
+    [episode, isFavorite],
+  )
+
+  const handlePressFavorite = () => {
+    onPressFavorite()
+    liked.value = withSpring(liked.value ? 0 : 1)
+  }
+
+  const handlePressCard = () => {
+    openLinkInBrowser(episode.enclosure.link)
+  }
+
   return (
+    //  MAVERICKTODO: Switch this out for Card component when that is ready
     <TouchableOpacity
       style={[$rowLayout, $item]}
-      onPress={() => openLinkInBrowser(episode.enclosure.link)}
+      onPress={handlePressCard}
+      onLongPress={handlePressFavorite}
+      // MAVERICKTODO: This button role should be set on the Card Component
+      accessibilityRole="button"
```

Comment:
Are we going with long press here because the favorite button is nested in a pressable?

This works, but I think a simpler design for screen readers that wouldn't affect visuals would involve _not_ nesting buttons. The majority of the card area could still be a button to listen, but the favorite icon button has its own separate pressable area. Thoughts?

Reply:
Yes, that's exactly the reason for it. I'm not sure how that would work. I know @yulolimum is working on a `<Card />` component, which I was planning to incorporate in this once it was finished, but I'm not sure how flexible the touchable area would be on that. Any thoughts @yulolimum?

Reply:
>but I'm not sure how flexible the touchable area would be on that

Generally I have been trying to expose customizations to all sub-components via a general prop similar to this: https://github.com/infinitered/ignite/blob/maverick/boilerplate/app/components/ListItem.tsx#L51-L54 . Or, as a spread for the wrapper: https://github.com/infinitered/ignite/blob/maverick/boilerplate/app/components/ListItem.tsx#L14.

It'll be the same for Card.

Reply:
I'll do a second PR integrating Card and I'll see if I can implement @lindboe's suggestion.